src/TodoList.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+
+let todoList = [{
```

Comment:
Nitpick (doesn't require changes):
This `todoList` can be defined as a `const` variable because it should never change. By assigning it as a `let` variable, it won't prevent you from accidentally redefining it somewhere within this file, which could cause bugs and unintended behavior.